torch/_dynamo/variables/torch.py

```diff
@@ -524,20 +478,11 @@ def fn_with_prim_types(x):
                     *proxy_args_kwargs([args[0]], {}),
                 ),
             )
-        elif self.value == torch.nn.init._calculate_correct_fan:
-            return UserFunctionVariable(
-                torch.nn.init._calculate_correct_fan
-            ).call_function(tx, args, {})
```

Comment:
```torch.nn.init._calculate_correct_fan``` is not torch in graph function, and it will go through ```skipfiles.check``` rule which returns inlining. So we should (and safe) remove it from here.
There are unit test in ```test_functions.py``` to make sure no regression for this.